website/docs/cloud-docs/integrations/kubernetes/index.mdx

```diff
@@ -1,68 +1,163 @@
 ---
-page_title: Terraform Cloud Operator for Kubernetes setup instructions
+page_title: Terraform Cloud Operator for Kubernetes 
 description: >-
   Terraform Cloud Operator allows you to provision infrastructure directly from
   the Kubernetes control plane.
-tfc_only: true
 ---
 
-# Terraform Cloud Operator for Kubernetes Setup Instructions
-
-## Overview
+# Terraform Cloud Operator for Kubernetes Overview
 
 HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official [Terraform Cloud Operator for Kubernetes](https://github.com/hashicorp/terraform-k8s) to provision infrastructure internal or external to the Kubernetes cluster directly from the Kubernetes control plane.  Using the Terraform Cloud Operator for Kubernetes' CustomResourceDefinition (CRD), users can dynamically create Terraform Cloud workspaces using a Terraform configuration from a git repository or from the [Terraform Registry](https://registry.terraform.io/), populate variables, and perform Terraform runs to provision infrastructure.
```

Comment:
`[Terraform Cloud Operator for Kubernetes](https://github.com/hashicorp/terraform-k8s)` refers to v1, do we want to update this link to v2 and mention v1 as legacy?

Reply:
Good catch! Updated.